pkg/apis/core/types.go

```diff
@@ -2124,6 +2134,11 @@ type Lifecycle struct {
 	// or until the termination grace period is reached.
 	// +optional
 	PreStop *Handler
+	// Type describes the startup/shutdown behaviour of the container.
+	// One of Standard, Sidecar.
```

Comment:
please quot "Standard" and "Sidecar" here.

What happens if a user sets this on an init container?

Reply:
You'll get an API validation error as 'Lifecycle' is not supported on init containers. That aside nothing would currently happen. Do you think this should be mentioned here?

Reply:
yes